.eslintrc.js

```diff
@@ -10,15 +10,16 @@ module.exports = {
   "env": {
     "es6": true,
     "node": true,
-    "browser": true
+    "browser": true,
+    "mocha": true
   },
   "rules": {
     "arrow-body-style": "off", // we don't something like (e => e.y) because it makes debugging harder
     "class-methods-use-this": "error",
     "consistent-return": "off",
-    "curly": "error",
+    curly: "error",
     "func-names": "error",
-    "func-style": ["error", "declaration", {"allowArrowFunctions": true}],
+    "func-style": ["error", "declaration", { "allowArrowFunctions": true }],
```

Comment:
Make sure the style is consistent, so all keys should be in quotes.

Reply:
I made sure that all keys are in quotes. Earlier a plugin in my editor was automatically removing the quotes from keys when the file was saved. I disabled that to make sure quotes are not removed